config/credentials.yml.enc

```diff
@@ -1 +1 @@
-QeK8qjnv/fDA2mX4zE7UrV65Xp54q/T/cv2y0DaJH7QJ9RtrpWa2o2HB5PlrHPAib0sYzSq0/jVOM/AMbeLeR6x7wGl8LRf3vNeHhDppFP4npx/OJAXArEHFtLJJow7gfdM6oHxDyAaVBi17I8qK8WghbRYqKJG536LUWpJ5d4yhfRcZ78oENFM2bikcCu5HY2IsXScPMXyF0gFJaPCnKmFV715NzF4qUhJXl2n3QG3SbcReip9hGCstqgTKOu9idfug5xvb0/mP44I8yuLzmZyPHTorIj50gq89vajMR7cHGoBzpNRxD8DAukUfWfDiYVkciakqmrOoWGr6J38OhOPvbgTvq8GxMqgp8z8p8DhACi8TsgUiBkBkfiGw+BKKdzkkHpwDl1fobEfsoRl8gtUUVq3yVqPqNENx--H1c4Hrtb/8Qg80C5--Jh94JUPB05Rq4Djqt0K5Rg==
\ No newline at end of file
+u/2eMMcp7jgR7jAu1yWfK/3+1s+t+xeQ2Ok6bbbOwOYSP41NKEsnka6yXqnMQAcLwiWTGxcaXl4uxgbALNbSkJDRnZF8aUDjhW4F9U7MTZgaGtD9/nQPNxHy/BPsKIBF9onSyam5Tae6REeHU3AjayMZsFukYVBh/xuMV+DTDZwcN222h+hVDCRzSYrGUK5lk2VUz3UK5UlG4wHE6FTS+AZPEzLbNOeqMDTf8H04T9PooRTV3j0WIBxqYT9Xon/CJShco4FUDAynH3pCMT2VAI1rnQ/rUupNPsSew4FPJ3jdRUixfoh1ZxZNncY8xSOBNizBHAw+82TZOriOgJaGkwAp985TLJkKl0rqs89+MXF4sJKSZQDF9oHfEq5asspRtfl0bMpJsTjRmfxRtDzi2acdUke7/cqUDs1z--gnNevISNOsI7VFt2--gw5JaFxQOOEmSFvHH4wUNA==
```

Comment:
Another suggestion was to update this file.  Now, on local, running `rake assets:precompile` works, so hopefully it'll work on Heroku as well.